tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1,6 +1,414 @@
 /* BEGIN_HEADER */
 #include "test/drivers/test_driver.h"
 
+/* Auxiliary variables for pake tests.
+   Global to silent the compiler when unused. */
+size_t pake_expected_hit_count = 0;
+int pake_in_driver = 0;
+
+#if defined(PSA_WANT_ALG_JPAKE)
+static void ecjpake_do_round(psa_algorithm_t alg, unsigned int primitive,
```

Comment:
What about to have two smaller functions: ecjpake_do_first_round() and then ecjpake_do_second_round() ?

Reply:
This function is taken from `psa_crypto_pake.function`, so I suggest to keep them in sync for now.

When we decide to simplify the state machine for pake operation this will have to be re-written.

Reply:
Yes sure, let's keep them in sync.